src/commands/lib/logout.ts

```diff
@@ -3,11 +3,14 @@ import { Profile, deleteUser } from "../../util/profile";
 import { out } from "../../util/interaction";
 
 export async function logout(client: MobileCenterClient, user: Profile): Promise<void> {
-  try {
-  await out.progress("Logging out current user...",
-    clientCall(async cb => client.account.deleteApiToken(await user.accessTokenId, cb)));
-  } catch (err) {
-    // Noop, it's ok if deletion fails
+  // Only delete token off the server if CLI created it.
+  if (!user.tokenSuppliedByUser) {
+    try {
+    await out.progress("Logging out current user...",
+      clientCall(async cb => client.account.deleteApiToken(await user.accessTokenId, cb)));
+    } catch (err) {
+      // Noop, it's ok if deletion fails
+    }
```

Comment:
Will this still print an error to the terminal or what will happen in case of error?